packages/lib/src/select/Select.tsx

```diff
@@ -624,6 +629,8 @@ const ClearSearchAction = styled.button`
   color: ${(props) => props.theme.actionIconColor};
   font-size: 1rem;
   cursor: pointer;
+  width: 100%;
```

Comment:
I think these extra values are not needed. I removed them and it looks correct.

Reply:
From my tests, this width is required, otherwise the icons won't be centered in the wrapper.
![image](https://github.com/user-attachments/assets/4e919348-fa34-49a0-8cf3-9b6a16a92a71)

Reply:
The styles I'm suggesting removing are from the searchable input's clear action. What you show me is from the clear selected options!

Reply:
You are absolutely right! Sorry for the confusion.